Wikipedia/Code/WMFExploreCollectionViewController.m

```diff
@@ -1080,9 +1080,6 @@ - (nonnull UICollectionReusableView *)collectionView:(UICollectionView *)collect
             return [collectionView dequeueReusableSupplementaryViewOfKind:UICollectionElementKindSectionFooter withReuseIdentifier:WMFFeedEmptyHeaderFooterReuseIdentifier forIndexPath:indexPath];
         case WMFFeedMoreTypeLocationAuthorization: {
             WMFTitledExploreSectionFooter *footer = (id)[collectionView dequeueReusableSupplementaryViewOfKind:UICollectionElementKindSectionFooter withReuseIdentifier:[WMFTitledExploreSectionFooter wmf_nibName] forIndexPath:indexPath];
-            footer.titleLabel.text = [EnableLocationViewController localizedEnableLocationTitle];
-            footer.descriptionLabel.text = [EnableLocationViewController localizedEnableLocationDescription];
-            [footer.enableLocationButton setTitle:[EnableLocationViewController localizedEnableLocationButtonTitle] forState: UIControlStateNormal];
```

Comment:
I think [Carolyn](https://phabricator.wikimedia.org/T164890#3341991) wanted the title to be "Explore articles near your current location", which is set in the storyboard, so deleting these lines make that storyboard string appear. So yeah we'll need new localized string for "Explore articles near your current location".